documentation/release_notes.markdown

```diff
@@ -16,6 +16,20 @@ Released 22 July 2019
 
 ### New features
 
+- This release includes an upgrade to the latest release of Jetty's 9.4 series. With this update, you may see "weak cipher" warnings about ciphers that were previously enabled by default. Puppet Server now defaults to stronger FIPS-compliant ciphers, but you must first remove the weak ciphers.
+
+The ciphers previously enabled by default have not been changed, but are considered weak by the updated standards. Remove the weak ciphers by removing the `cipher-suite` configuration section from the `webserver.conf`. After you do this, Puppet Server uses the FIPS-compliant ciphers instead. This release includes the weak ciphers for backward compatibility only.
```

Comment:
> After you do this, Puppet Server uses....

Should it be

> After doing this, Puppet Server uses

Or

> After you do this, Puppet Server will use

?

Reply:
I think it's correct as it is. "After doing this..." implies that Puppet Server is removing the weak ciphers, and the future tense isn't necessary in the second possibility.

After the user does this thing, Puppet Server will use, and will continue to use on a regular basis, the new ciphers.

All that said, "after you do this" is a little lazy/rushed writing, so I'm making it more explicit.